components/navbar.py

```diff
@@ -223,16 +234,86 @@ def my_navbar():
 def toggle_navbar_collapse(is_open, burger_state, *args):
     trigger = ctx.triggered_id
 
-    # When the toggler is clicked, toggle the collapse state
+    # Set default style with padding-top 60px
```

Comment:
remove this styles and add them inline with the component as discussed yesterday